index.js

```diff
@@ -22,6 +22,19 @@ module.exports = (api, projectOptions) => {
     if (options.customRendererConfig) {
       Object.assign(rendererConfig, options.customRendererConfig);
     }
+    
+    const renderer = new Renderer(rendererConfig);
+    if (projectOptions.baseUrl) {
+      renderer.preServer = (Prerenderer) => {
```

Comment:
I don't think I've seen this in the docs... Where do you get that from?

Reply:
@SolarLiner in the code https://github.com/Tribex/prerenderer/blob/master/es6/index.js#L61

Reply:
Hm. It seems they skipped that in the docs then. Thanks for the PR!